[TW-1273] Rewrite onboarding checklist (#5218)
postmanlabs/postman-docs: src/pages/docs/administration/onboarding-checklist.md

---
title: "Onboarding checklist"
updated: 2023-11-13
contextual_links:
  - type: section

>     name: "Additional resources"

Reviewer: I would remove the blog post here because it's pretty outdated

---

> To begin using Postman within your organization, you can perform the following tasks to set your Postman team up for success. It's recommended that you first collaborate with your identified Team Admins and your organization's IT team to set up, secure, and manage Postman in your organization. Then you can set up your Postman team by configuring relevant settings, inviting people to your team and assigning them roles, and creating workspace resources related to your projects.

Reviewer: The first sentence comes across as a bit formal to me, particularly "perform". Maybe could switch that with something like "walk through" or reword it slightly to something like 
"To get started using Postman within your organization, you can walk through the following tasks to set your Postman team up for success."

> If you're using the Postman web app, Postman recommends using the Postman Desktop Agent for the best experience. See [About the Postman Agent](/docs/getting-started/basics/about-postman-agent/) for more information.

## Contents

* [Set up Postman in your organization](#set-up-postman-in-your-organization)
    * [Manage Team Admins and support users](#manage-team-admins-and-support-users)
    * [Contact your IT team](#contact-your-it-team)
    * [Secure your Postman team](#secure-your-postman-team)
    * [Merge teams into a single account](#merge-teams-into-a-single-account)
    * [Manage the Enterprise app](#manage-the-enterprise-app)
* [Set up your Postman team](#set-up-your-postman-team)
    * [Configure team settings](#configure-team-settings)
    * [Manage team members](#manage-team-members)
    * [Create workspaces and collections](#create-workspaces-and-collections)
    * [Configure integrations](#configure-integrations)
* [Questions](#questions)

## Set up Postman in your organization

> To begin setting up Postman, assign the Team Admin role to one or more users, giving them permission to set up Postman and your team. Then Team Admins can contact your IT team for help with setting up, managing, and securing Postman in your organization. After you set up Postman, you can [set up your Postman team](#set-up-your-team).

Reviewer: ```suggestion
To begin setting up Postman, [assign the Team Admin role](/docs/administration/managing-your-team/managing-your-team/#managing-team-roles) to one or more users. This role gives them permission to set up Postman and your team. Team Admins can then contact your IT team for help with setting up, managing, and securing Postman in your organization. After you set up Postman, you can [set up your Postman team](#set-up-your-postman-team).
```

Author: Thanks! Suggestions looks good. I moved the link to `#managing-team-roles` to the next section.

    > If you're on a [Postman Enterprise plan](https://www.postman.com/pricing), your IT team can choose to deploy the Postman Enterprise app across your organization. For more information, see [Manage the Enterprise app](#manage-the-enterprise-app).

* If your organization's network connection is behind a proxy, you may need to configure Postman appropriately. Retrieve proxy connection details from your IT team and [​set them up within Postman](/docs/getting-started/installation/proxy/)​.

* If your organization operates behind a firewall, your IT team may need to [configure allowlists for Postman's domains](/docs/getting-started/installation/installation-and-updates/#use-postman-behind-a-firewall). This ensures Postman data is synced with the cloud and all functionality works as expected.

* Depending on your [plan](https://www.postman.com/pricing), you may be able to obtain static IP addresses for Postman Monitors, enabling you to monitor APIs behind a restricted firewall. Your IT team must [allowlist static IPs for monitoring](/docs/monitoring-your-api/using-static-IPs-to-monitor/#allowlisting-static-ip-addresses).

### Secure your Postman team

> Depending on your [plan](https://www.postman.com/pricing), you can secure your Postman team's users by setting up [SSO](/docs/administration/sso/intro-sso/), [SCIM](/docs/administration/scim-provisioning/scim-provisioning-overview/), and domain [verification](/docs/administration/domain-verification-and-capture/add-and-verify-a-domain/#why-should-my-organization-verify-a-domain-with-postman) and [capture](/docs/administration/domain-verification-and-capture/domain-capture-overview/#why-should-my-organization-enable-domain-capture-in-postman). You must first configure SSO before you can configure SCIM provisioning and domain capture.

Reviewer: ```suggestion
Depending on your [plan](https://www.postman.com/pricing), you can secure your Postman team by setting up [SSO](/docs/administration/sso/intro-sso/), [SCIM](/docs/administration/scim-provisioning/scim-provisioning-overview/), and domain [verification](/docs/administration/domain-verification-and-capture/add-and-verify-a-domain/#why-should-my-organization-verify-a-domain-with-postman) and [capture](/docs/administration/domain-verification-and-capture/domain-capture-overview/#why-should-my-organization-enable-domain-capture-in-postman). You must first configure SSO before you can configure SCIM provisioning and domain capture.
```

Team Admins can [configure SCIM (System for Cross-domain Identity Management) for your team](/docs/administration/scim-provisioning/scim-provisioning-overview/#enabling-scim-provisioning) to automate user provisioning and de-provisioning for your Postman team using your IdP, such as Okta or OneLogin.

Team Admins can [configure domain capture](/docs/administration/domain-verification-and-capture/enable-domain-capture/) to consolidate Postman user accounts in your organization into a single team. Before you configure domain capture, it's recommended that you notify users that their accounts will be merged, and [enable Auto-flex](/docs/billing/billing/#using-auto-flex) to ensure your team can automatically accommodate all users added to your team.

### Merge teams into a single account

> You can merge one or more Postman teams into a single, company-authorized Postman account. This enables users in your organization to collaborate in the same Postman team. To merge Postman teams, you must [migrate team data into the company-authorized account](/docs/administration/team-merge/).

Reviewer: I would add a note here mentioning to contact support to figure out the best migration plan for your org since the merge doc is incomplete/outdated

You can also [manage your team profile](/docs/administration/managing-your-team/team-settings/#managing-your-team-profile) to share information about your team with other Postman users. This ensures your API's consumers have important details about your team's public resources.

### Manage team members

Team Admins can [manage team members](/docs/administration/managing-your-team/managing-your-team/#managing-team-members) by adding, removing, and assigning roles to users. You can also organize team members into groups, and assign roles to groups instead of individual users, enabling you to efficiently onboard new team members.

You can [invite users to your team](/docs/administration/managing-your-team/managing-your-team/#inviting-users) by generating a shareable link or by inviting them through a direct email.

> You can [manage team-level roles](/docs/administration/managing-your-team/managing-your-team/#managing-team-roles) assigned to individual team members, enabling them to perform different actions within your team. You can also [organize team members into groups](/docs/collaborating-in-postman/user-groups/), and assign team-level roles to groups. Learn more about [team-level roles](/docs/collaborating-in-postman/roles-and-permissions/#team-roles).

Reviewer: ```suggestion
You can [assign team-level roles](/docs/administration/managing-your-team/managing-your-team/#managing-team-roles) to individual team members, enabling them to perform different actions within your team. You can also [organize team members into groups](/docs/collaborating-in-postman/user-groups/), and assign team-level roles to groups.

> Learn more about [team-level roles](/docs/collaborating-in-postman/roles-and-permissions/#team-roles).
```
maybe a note would stick out more?

In your workspaces, you can [group your requests and examples into collections](/docs/collections/using-collections/). This enables you to keep your workspaces organized by project, collaborate with teammates, run tests on related requests, and more. If you're not sure how to set up your collection, you can [create a collection from a template](/docs/collections/using-collections/#using-collection-templates). Postman has a variety of collection templates you can use for your projects.

> You can assign [workspace roles]((/docs/collaborating-in-postman/roles-and-permissions/#workspace-roles)) to team members and groups at the [workspace level](/docs/collaborating-in-postman/using-workspaces/managing-workspaces/#managing-workspace-roles). You can also assign [element-based roles](/docs/collaborating-in-postman/roles-and-permissions/#element-based-roles) at the [element level](/docs/collaborating-in-postman/requesting-access-to-elements/#managing-element-based-roles), such as collections and APIs.

Reviewer: ```suggestion
You can assign [workspace roles](/docs/collaborating-in-postman/roles-and-permissions/#workspace-roles) to team members and groups at the [workspace level](/docs/collaborating-in-postman/using-workspaces/managing-workspaces/#managing-workspace-roles). You can also assign [element-based roles](/docs/collaborating-in-postman/roles-and-permissions/#element-based-roles) at the [element level](/docs/collaborating-in-postman/requesting-access-to-elements/#managing-element-based-roles), such as collections and APIs.
```

> Depending on your [plan](https://www.postman.com/pricing), you may be able to [manage who can create team workspaces](/docs/administration/managing-your-team/managing-your-team/#manage-your-teams-workspaces) in your team.

Reviewer: ```suggestion
Depending on your [plan](https://www.postman.com/pricing), you may also be able to [manage who can create team workspaces](/docs/administration/managing-your-team/managing-your-team/#manage-your-teams-workspaces) in your team.
```

### Configure integrations

Postman has multiple integrations for popular third-party solutions. You can [add integrations to your team](/docs/integrations/intro-integrations/) to automatically share data between Postman and other tools, sync your collections and API definitions, and more.

## Questions

> If you have any questions or run into any issues setting up Postman for your team, check out the [Postman Community](https://community.postman.com/). You can find a number of support resources on the [support page](https://www.postman.com/support). You can also contact [Postman support](https://www.postman.com/support/) with any questions.

Reviewer: ```suggestion
If you have any questions or run into any issues setting up Postman for your team, check out the [Postman support center](https://www.postman.com/support) or explore the [Postman Community](https://community.postman.com/). You can also contact [Postman support](https://www.postman.com/support/) with any questions.
```
I think we probably want to mention the support center first as that is a bit more geared towards teams/admins, whereas the community is mostly technical questions

Author: Yes, makes sense!
